src/rp2040/boot_stage2/include/boot_stage2/config.h

```diff
@@ -11,6 +11,8 @@
 
 #include "pico.h"
 
+// PICO_CONFIG: PICO_FLASH_SPI_CLKDIV, Clock divider from clk_sys to use for serial flash communications in boot stage 2. On RP2040 this must be a multiple of 2, type=int, default=varies; often specified in board header, advanced=true, group=boot_stage2
```

Comment:
This file is in `src/rp2040/...` so is already RP2040-specific, so the "On RP2040 " can probably be removed?